src/Shell/Task/QueueEmailTask.php

```diff
@@ -78,7 +78,7 @@ public function run(array $data, $jobId) {
 		if (is_object($email) && $email instanceof Email) {
 			try {
 				$transportClassNames = $email->configuredTransport();
```

Comment:
Is this still used then?

Reply:
And wouldnt it be more BC to check on the defined transport, and if not fall back to the current one?

Reply:
You're right, the `$transportClassNames` is not used anymore. 

I agree it is safer to check the existence of the transport property. But you are executing it in a try..catch block. The check for an existing (correct) transport defenition should be done in the cakephp core I would say. 
However, we could at least check for the existence of the property, like you did in line 71?

Reply:
we could, yeah